docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,301 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # Demo
</pre><p>remove comment its not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>